### PR TITLE
Navbar current page highlight

### PR DESCRIPTION
**Pull request by jkempster34**
### Trello card
[567](https://trello.com/c/DJDXN592/567-develop-move-funding-after-steps-in-top-nav-bar-and-add-highlight-to-page-title-in-nav-when-you-are-on-that-page)

### Context
background highlight on current navbar links

### Guidance to review
This was reverted yesterday as I forgot to update the mobile links to reflect the changes in the content. I have added some tests to prevent that happening again

### CODE REVIEW

File: app/views/sections/_header.html.erb [hunk 1]
```diff
@@ -8,5 +8,5 @@
             <a href="#" data-action="click->banner#hide">Hide this message</a>
         </p>
     </section>
-    <%= render(Navbar::NavbarComponent.new(@sitemap)) %>
+    <%= render(Navbar::NavbarComponent.new()) %>
```
Comment (peteryates) [inline app/views/sections/_header.html.erb:11]: You don't need the empty parentheses here, just `.new` is sufficient.

**Review by peteryates — APPROVED** [review]
Looks good

File: app/components/navbar/navbar_component.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <a data-action="click->navigation#navToggle" href="javascript:void(0);" class="icon">
+                <div data-target="navigation.hamburger" id='hamburger' class="icon-close"></div>
+                <div data-target="navigation.label" id="navbar-label" class="icon-hamburger-label">Close</div>
+            </a>
+        </div>
+        <div data-target="navigation.links" id="navbar-mobile-links" class="navbar__mobile__links">
+            <ul>
+                <li><a href="/">Home</a></li>
+                <% resources.each do |resource| %>
+                    <li><a href="<%= resource[:path] %>"><%= resource[:front_matter][:title] %></a></li>
```
Comment (peteryates) [inline app/components/navbar/navbar_component.html.erb:26]: Shoulld probably be using a `link_to` here rather than manually building the `a` element

**Review by peteryates — CHANGES_REQUESTED** [review]
Looks fine other than the minor things I mentioned

**Comment by jkempster34** [comment]
> Looks fine other than the minor things I mentioned

Made those changes

**Review by peteryates — APPROVED** [review]
Looks great!

**Pull request merged**